src/components/home/Home.js

```diff
@@ -1,10 +1,46 @@
-import React from 'react';
-import './home.css';
-
-const Home = () => (
-  <div>
-    <h1>Home page</h1>
-  </div>
-);
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getDoctors } from '../../redux/doctors/doctorsSlice';
+
+function Home() {
+  const { doctors, isLoading, error } = useSelector((store) => store.doctors);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getDoctors());
+  }, [dispatch]);
+
+  if (isLoading) {
+    return (
+      <div className="loading">
+        <div className="spinner" />
+      </div>
+    );
+  }
+
+  if (error) {
+    return (
+      <div className="error-container">
+        <h2>Oopps somethings went wrong.PLease try again!</h2>
+        <p>{error}</p>
+      </div>
+    );
+  }
+  return (
+    <div className="home-page">
+      {doctors.map((doctor) => (
+        <div className="doctor-info" key={doctor.id}>
+          <img className="photo" src={doctor.photo} alt={doctor.name} />
+          <h2>{doctor.name}</h2>
+          <p>{doctor.about}</p>
+          <p>
+            Buy one hour of time with only $
+            {doctor.price_hour}
+          </p>
+        </div>
+      ))}
+    </div>
```

Comment:
You are displaying the time as a whole string, but that is not a good user experience. You should remove the unnecessary part of that string. You could also change the number of the month to its name, making it easier for the user to understand.
You could try something like 23 June, 2023.

Reply:
Hi @Muhammad0602,
thanks for your good and valuable feedback, your suggestions are applied.
![Capture2](https://github.com/Iqbal-Elham/Doctor-Reservation-frontend/assets/108926875/5366a729-4206-472c-9b38-81c55d8f0a76)